src/services/HapticService.ts

```diff
@@ -24,6 +24,6 @@ function triggerHaptic(hapticMethod: string, componentName: string) {
 }
 
 export default {
-  HapticMethods,
+  HapticType,
```

Comment:
No migration needed?

Reply:
Nope, it's a new code so it's not in use yet :)